src/libraries/Microsoft.Extensions.Hosting/src/HostBuilder.cs

```diff
@@ -121,6 +121,7 @@ public IHostBuilder ConfigureContainer<TContainerBuilder>(Action<HostBuilderCont
         /// Run the given actions to initialize the host. This can only be called once.
         /// </summary>
         /// <returns>An initialized <see cref="IHost"/></returns>
+        /// <remarks>Adds basic services to the host such as application lifetime, host environment and logging.</remarks>
```

Comment:
Microsoft uses Oxford commas.

```suggestion
        /// <remarks>Adds basic services to the host such as application lifetime, host environment, and logging.</remarks>
```